### PR TITLE
General improvements after release for node.

### PR DESCRIPTION
**Pull request by neal**
@matthewtole

### CODE REVIEW

File: node/app.js [hunk 1]
```diff
@@ -5,107 +5,77 @@ var Timeline = require('pebble-api');
 var app = express();
 app.set('port', (process.env.PORT || 5000));
 
+// create a new Timeline object with our API key keys passed as an enviornment variable
 var timeline = new Timeline({
   apiKey: process.env.PEBBLE_TIMELINE_API_KEY
```
Comment (matthewtole) [inline node/app.js:10]: You should make it throw an error if you haven't set this environment variable.

**Comment by matthewtole** [comment]
Aside from above comment, :+1:

File: node/app.js (hunk 1 above)
Comment (neal) [inline node/app.js:10]: It would just pass `undefined` and the library would throw an error eventually.

Comment (matthewtole) [inline node/app.js:10]: Didn't for me!

Comment (matthewtole) [inline node/app.js:10]: Never mind, it did.

**Pull request merged**